app/views/homepage/index.html.erb

```diff
@@ -109,29 +109,21 @@
 
         <div class="govuk-grid-column-one-third">
           <a href="/transition" aria-hidden="true" tabindex="-1">
-            <%= image_tag "homepage/time-is-running-out--480w.jpg", {
+            <%= image_tag "homepage/transition-period.png", {
               alt: "",
               class: "home-promo__image",
               height: 400,
               loading: "lazy",
-              width: 600,
-              sizes: "(max-width: 640px) 100vw, (max-width: 1020px) 33vw, 300px",
-              srcset: {
-                "homepage/time-is-running-out--600w.jpg": "600w",
```

Comment:
no srcset for this one?
Is this less optimised for mobile?

Reply:
Removed it because the image we have is ~5kb - we might add in various sized variants if we need to afterwards